kubernetes/decoupled/autoscaler-pkg/poller/poller-hourly.yaml

```diff
@@ -45,6 +45,8 @@ spec:
                       fieldPath: metadata.name
                 - name: OTLP_COLLECTOR_URL
                   value: "http://otel-collector:4317/"
+                - name: OLTP_IS_LONG_RUNNING_PROCESS
```

Comment:
`OLTP_` -> `OTLP_` throughout. Strictly I suppose `OTEL` would be more accurate here (to refer to OpenTelemetry in general as opposed to its associated protocol) and potentially also in the entry above for consistency, but I don't think this is a show-stopper unless you are keen to update.

Reply:
Done